cmd/config.go

```diff
@@ -154,12 +161,28 @@ func (c *clientConfig) Check() error {
 	if c.TUN.Timeout != 0 && c.TUN.Timeout < 4 {
 		return errors.New("invalid TUN timeout")
 	}
+	if len(c.TCPRelay.Listen) > 0 && len(c.TCPRelay.Remote) == 0 {
+		return errors.New("no TCP relay remote address")
+	}
+	if len(c.UDPRelay.Listen) > 0 && len(c.UDPRelay.Remote) == 0 {
+		return errors.New("no UDP relay remote address")
+	}
```

Comment:
for the check logic , I have an idea, we can use the [github.com/go-playground/validator](https://github.com/go-playground/validator) to validate the field, anyway we can do it in other commit